test/schema.test.js

```diff
@@ -1696,57 +1696,57 @@ describe('schema', function() {
     });
 
     it('returns the schema instance', function() {
-      const ret = this.schema.clone().remove('g');
+      const ret = this.$__schema.clone().remove('g');
       assert.ok(ret instanceof Schema);
     });
 
     it('removes a single path', function(done) {
-      assert.ok(this.schema.paths.a);
-      this.schema.remove('a');
-      assert.strictEqual(this.schema.path('a'), undefined);
-      assert.strictEqual(this.schema.paths.a, void 0);
+      assert.ok(this.$__schema.paths.a);
```

Comment:
Tests should still use `this.schema`, not `this.$__schema`. `this.$__schema` is for internal use in case where the user decides to add a `schema` path to their schema, but we should still make sure `schema` works.